Maths/Factorial.php

```diff
@@ -1,30 +1,34 @@
 <?php
+
 /**
  * This function calculates
  * and returns the factorial
  * of provided positive integer
  * number.
  *
- * @param Integer $number Integer input
+ * @param  Integer  $number  Integer input
  * @return Integer Factorial of the input
+ * @throws \Exception
  */
 function factorial(int $number)
 {
-    static $cache = [];//internal caching memory for speed
+    static $cache = [];
+//internal caching memory for speed
```

Comment:
```suggestion
    static $cache = []; //internal caching memory for speed
```

Reply:
Or could do this too - I just don't like having the comment below

```suggestion
    //internal caching memory for speed
    static $cache = [];
```